contrib/gitian-descriptors/gitian-linux.yml

```diff
@@ -2,31 +2,34 @@
 name: "bitcoin-linux-0.11"
 enable_cache: true
 suites:
-- "precise"
+- "trusty"
 architectures:
 - "amd64"
-packages: 
+packages:
 - "g++-multilib"
 - "git-core"
 - "pkg-config"
-- "autoconf2.13"
+- "autoconf"
 - "libtool"
 - "automake"
 - "faketime"
 - "bsdmainutils"
 - "binutils-gold"
-- "libstdc++6-4.6-pic"
-reference_datetime: "2015-06-01 00:00:00"
+- "ca-certificates"
+- "python"
 remotes:
 - "url": "https://github.com/bitcoin/bitcoin.git"
   "dir": "bitcoin"
 files: []
 script: |
   WRAP_DIR=$HOME/wrapped
   HOSTS="i686-pc-linux-gnu x86_64-unknown-linux-gnu"
-  CONFIGFLAGS="--enable-glibc-back-compat --enable-reduce-exports LDFLAGS=-static-libstdc++"
+  CONFIGFLAGS="--enable-glibc-back-compat --enable-reduce-exports --disable-bench --disable-gui-tests"
```

Comment:
I accidentally left `--disable-bench` in here (upstream added a benchmark framework in bitcoin/bitcoin#6733 with an associated disable flag that they added to gitian), but it does no harm and will mean we don't forget about adding it if we do eventually pull in bitcoin/bitcoin#6733 from upstream.